homeassistant/components/tuya/config_flow.py

```diff
@@ -379,6 +384,17 @@ def _get_climate_schema(curr_conf, device):
                     CONF_CURR_TEMP_DIVIDER,
                     default=curr_conf.get(CONF_CURR_TEMP_DIVIDER, 0),
                 ): vol.All(vol.Coerce(int), vol.Clamp(min=0)),
+                vol.Optional(
+                    CONF_PRECISION_OVERRIDE,
+                    default=curr_conf.get(CONF_PRECISION_OVERRIDE, CONF_DEFAULT),
+                ): vol.In(
+                    {
+                        PRECISION_TENTHS: "Tenths",
+                        PRECISION_HALVES: "Halves",
+                        PRECISION_WHOLE: "Whole",
+                        CONF_DEFAULT: "Default",
```

Comment:
Why `CONF_DEFAULT` as last option in the list and not first?

Reply:
You're right. I added it at first position.